media/js/base/mozilla-fxa-form.js

```diff
@@ -102,6 +102,50 @@ if (typeof window.Mozilla === 'undefined') {
         });
     };
 
+
+    /**
+     * Intercept event handler for FxA links and forms, lets the browser drive the FxA Flow using
+     * the `showFirefoxAccounts` UITour API. Attaches several UTM parameters from the current page
+     * that will be forwarded to the browser and later on to FxA services.
+     * @param event {Event}
+     * @private
+     */
+    FxaForm.interceptFxANavigation = function(event) {
+        event.preventDefault();
+        var urlParams = new window._SearchParams().utmParams();
+        var attributionData = Mozilla.UtmUrl.getAttributionData(urlParams);
```

Comment:
Looks like we already have a `FxaForm.getUTMParams()` function above on line 24. Rather than repeat the logic here, might it be worth altering that function to return only the params, and then the calling code can reuse it for each context? That way we'd only have to query the params once, rather than do it all again here.